www/src/context/SettingsContext.jsx

```diff
@@ -23,13 +23,18 @@ const SettingsContextProvider = ({ children }) => {
     document.body.setAttribute('dir', e.target.value);
     setDirection(e.target.value);
     global.localStorage.setItem('pgn__direction', e.target.value);
-    window.analytics.track('Direction change');
+    global.analytics.track(`Direction change (${e.target.value})`);
```

Comment:
I think it would make sense to pass these values as event properties rather than a distinct event, e.g.:

```js
global.analytics.track('Direction change', { direction: e.target.value });
```

This would be similar feedback for most of the other events as well, e.g. "Toggle settings" might pass a `value` property.

Reply:
Added event properties for all occurrences